requirements.txt

```diff
@@ -1,3 +1,4 @@
 xmlschema>=1.6
 tifffile>=2020.10.1
 jsonschema==4.4.0
+git+https://github.com/hubmapconsortium/fastq-utils.git@main#egg=hubmap-fastq-utils
```

Comment:
I recommend changing this to
```
git+https://github.com/hubmapconsortium/fastq-utils@v0.2.5
```
which contains the fix for https://github.com/hubmapconsortium/fastq-utils/issues/1.

Reply:
Done.